libs/metrics/prometheus/prometheus_test.go

```diff
@@ -248,3 +226,22 @@ func TestInconsistentLabelCardinality(t *testing.T) {
 		"a", "1", "b", "2", "c", "KABOOM!",
 	).Add(123)
 }
+
+func spinUpPrometheusServer() *httptest.Server {
+	s := httptest.NewServer(promhttp.HandlerFor(stdprometheus.DefaultGatherer, promhttp.HandlerOpts{}))
+
+	// minimal delay to allow the prometheus server come up with results and avoid errors during test
```

Comment:
What results? At this point, we have not registered any metrics.

Reply:
I think you are asking only about the comment right ? I meant just for the server to be up, not results, I can remove that from the comment